src/ui/activity-stream/Notification.js

```diff
@@ -2,11 +2,16 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage, FormattedRelative } from 'react-intl';
 import { Icon } from 'patternfly-react';
+import { isString } from 'lodash';
 import { Panel, Form, Button, Row, Col } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
 import { ROUTE_USER_DETAIL } from 'app-init/router';
 import { routeConverter } from '@entando/utils';
 
+const toMessageDescriptor = message => (
```

Comment:
what about using a separate react component instead of a function (that is duplicated in other files)?